_includes/about-table-contents.html

```diff
@@ -0,0 +1,20 @@
+<div class="about-table-contents">
+  <ul class="btn">
+    <li>Skip to...</li>
+    <li>
+      <a href="#">
+      Commission Members<i class="icon-angle-right"></i>
```

Comment:
Rather than including the icons in the HTML, it would be better to render them in psuedo elements via CSS. There's an example of how to do that on Trade Guys [here](https://github.com/CSIS-iLab/trade_guys/blob/master/assets/_sass/base/_typography.scss#L358-L369). That way we can add additional links if necessary. You can get the character code for the icon from the `_fontello.scss` file.